src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -435,6 +436,42 @@ async function validate_delete_account(config_root_backend, buckets_dir_path, ac
     });
 }
 
+    // TODO - when we have the structure of config we can check easily which IAM users are owned by the root account
+    // currently, partial copy from _list_config_files_for_users
+    async function check_if_root_account_does_not_have_IAM_users(
+        config_root_backend, accounts_dir_path, account_to_check) {
```

Comment:
try take it to the function signature line

Reply:
Moved to one line.